domains/amila.json

```diff
@@ -1,10 +1,10 @@
 {
     "owner": {
-        "username": "Y-AmilaNirmal",
+        "username": "y-amila-nirmal",
         "email": "yamilanirmal@gmail.com"
     },
 
     "record": {
-        "CNAME": "y-amilanirmal.github.io"
+        "CNAME": "y-amila-nirmal.github.io"
```

Comment:
404 GitHub page. Please have an actual site to display.

Reply:
Previously, the site was unpublished for some reason.
But now everything is working. I thank you for helping me with this. thanks 😁😁😁